controllers/datafoundation.go

```diff
@@ -114,3 +120,21 @@ func (r *dataFoundationReconciler) reconcileOnboardingValidationSecret() error {
 	}
 	return nil
 }
+
+func (r *dataFoundationReconciler) reconcileCephIngressNetworkPolicy() error {
+	cephIngressNetworkPolicy := &netv1.NetworkPolicy{}
+	cephIngressNetworkPolicy.Name = "ceph-ingress-rule"
+	cephIngressNetworkPolicy.Namespace = r.managedFusionOffering.Namespace
```

Comment:
The current convention in the codebase it to initialize the identity of the reconciled resources in init function. Please follow the guideline

Reply:
Updated